.circleci/config.yml

```diff
@@ -1,7 +1,7 @@
 version: 2
 
 jobs:
-  build:
+  "macOS":
```

Comment:
no need to wrap these in quotes

Reply:
I wanted to indicate that these are chosen strings vs. stuff that from a list Circle expects. Bad idea?

Reply:
They're both the same when they get parsed anyway :woman_shrugging:, as much as I wish YAML required quotes everywhere, stylistically, I'm not a fan of them when they're not required, as it makes me assume something weird is going on.